### ISSUE TITLE
Flaky test

### ISSUE THREAD

**Issue by fulmicoton** [quickwit-oss/quickwit#2347]
I am not sure if it is already known:

https://github.com/quickwit-oss/quickwit/actions/runs/3489870921/jobs/5840546515

```
metastore::grpc_metastore::common_tests::test_metastore_list_splits ---
thread 'metastore::grpc_metastore::common_tests::test_metastore_list_splits' panicked at 'assertion failed: `(left == right)`
  left: `{"list-splits-six", "list-splits-two", "list-splits-five", "list-splits-four", "list-splits-three", "list-splits-one"}`,
 right: `{"list-splits-one", "list-splits-two", "list-splits-five", "list-splits-four", "list-splits-three"}`', quickwit-metastore/src/tests.rs:2114:13
stack backtrace:
```

**Comment by ChillFish8** [comment]
Known and PR open to fix #2324

**Comment by fulmicoton** [comment]
sweeet thank you!
